app/controllers/api/connections_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::ConnectionsController < ApplicationController
+    include AuthenticationConcern
+
+    def request_match
+
+        if @current_user
+
+            request = MatchRequest.new
+            request.student_id = @current_user.id
+
+            if SkillClass.find_by(id: params[:id])
```

Comment:
Here, you can probably move the contents of lines 45-48 to another method and replace this with `<new_method_name> and return unless SkillClass.find_by(id: params[:id])`. Alternatively, since `params[:id]` is present from the very beginning, you can also move this verification to a `before_action` callback → if the `SkillClass` doesn't exist, you don't even have to initialize a `MatchRequest` at all